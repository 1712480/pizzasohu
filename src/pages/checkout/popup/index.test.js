import React from 'react';
import renderer from 'react-test-renderer';
import Popup from '.';

it('Popup snapShot', () => {
    let snap = renderer.create(
        <Popup />
    ).toJSON();

    expect(snap).toMatchSnapshot();
})