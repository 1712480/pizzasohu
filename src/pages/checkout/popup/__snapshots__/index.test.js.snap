// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup snapShot 1`] = `
<div
  className="modal"
  id="modal-name"
>
  <div
    className="modal-box"
  >
    <div
      className="modal-header"
    >
      <p>
        Confirmation
      </p>
    </div>
    <div
      className="modal-body"
    >
      <p>
        Submit order
      </p>
      <br />
      <button
        onClick={[Function]}
      >
        Yes
      </button>
      <button
        className="no"
        onClick={[Function]}
      >
        No
      </button>
    </div>
  </div>
</div>
`;
